### PR TITLE
Popup: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

**Comment by ksercs** [comment]
There is `_F:\devextreme-documentation\concepts\05 Widgets\Tooltip\10 Customize the Content\Customize the Content.md_` empty file. Should we remove it or add some tags?

File: concepts/05 Widgets/Popup/05 Customize the Appearance/10 Customize the Title.md [hunk 1]
```diff
@@ -32,6 +32,70 @@ By default, the **Popup** allocates a part of its area to the title, regardless
...
+            this.setState({
+                isPopupVisible: false
+            });
+        }
+
+        render() {
+            return (
+                <Popup
+                    visible={this.state.isPopupVisible}
+                    showTitle={false}
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Popup/05 Customize the Appearance/10 Customize the Title.md:91]: onHiding is missing.

File: concepts/05 Widgets/Popup/05 Customize the Appearance/20 Specify Toolbar Items.md [hunk 2]
```diff
@@ -67,6 +67,110 @@ The **Popup** has two toolbars: top and bottom. Items on these toolbars can be p
...
+            });
+        }
+
+        render() {
+            return (
+                <Popup
+                    visible={this.state.isPopupVisible}
+                    title="Popup Title"
+                    onHiding={this.onHiding}
+                >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Popup/05 Customize the Appearance/20 Specify Toolbar Items.md]:
```suggestion
                    onHiding={this.onHiding}>
```

File: concepts/05 Widgets/Popup/10 Show and Hide the Popup/01 API.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ To show or hide the **Popup** programmatically, call the [show()](/api-reference
 
 ---
 
-With Angular, AngularJS or Knockout, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#visible') property of the **Popup** widget to a component property (in Angular), a scope property (in AngularJS), or an observable variable (in Knockout). After that, change them, and the **Popup** will appear or disappear.
+With Angular, Vue or React, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#visible') property of the **Popup** widget to a component property. After that, change them, and the **Popup** will appear or disappear.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Popup/10 Show and Hide the Popup/01 API.md]:
```suggestion
With Angular, Vue or React, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#visible') property of the **Popup** widget to a component property. After that, change this component property, and the **Popup** will appear or disappear.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Popup/05 Customize the Appearance/10 Customize the Title.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/Popup/05 Customize the Appearance/10 Customize the Title.md:91]: fixed

File: concepts/05 Widgets/Popup/05 Customize the Appearance/20 Specify Toolbar Items.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/Popup/05 Customize the Appearance/20 Specify Toolbar Items.md]: fixed

File: concepts/05 Widgets/Popup/10 Show and Hide the Popup/01 API.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/Popup/10 Show and Hide the Popup/01 API.md]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**